views/main.js

```diff
@@ -132,28 +132,35 @@ function render(data) {
     $answers.html('');
 
   // Back Button
-  if (actions.length) {
-    var $backBtn = $('<button class="back-button">Back</button>');
-    $backBtn.click(function() {
-      var previous = actions.pop();
-
-      var checklistIdx = checklist.indexOf(previous.checklist);
-      if (checklistIdx >= 0) {
-        checklist.splice(checklistIdx, 1);
-      }
-
-      render(previous);
-    });
+    if (actions.length) {
```

Comment:
@vmacdougal Can you specify on each change what has happened? 
It looks good visually but having the comments would be nice for clarity for people that aren't as good at dev and for our own documentation.